nototools/cldr_data.py

```diff
@@ -293,6 +293,13 @@ def is_script_rtl(script):
   try:
     return _SCRIPT_METADATA[script][5] == 'YES'
   except KeyError:
+    # special case a few codes and data we have that hasn't been
+    # updated.  Also special case locale-script codes, we have some.
+    if script == 'Adlm':
+      return True
+    if script in ['Zsym', 'Zsye', 'Hrkt', 'Jpan']:
```

Comment:
Good question.  I don't have a good answer, basically because I'm not running into them.  I wondered about this too but didn't look into it far enough.